docker/dokku/Dockerfile

```diff
@@ -1,39 +1,29 @@
-FROM python:alpine3.6
+FROM python:3.7-slim
```

Comment:
We could actually also try and go with `python:3.8-slim`, no?

```suggestion
FROM python:3.8-slim
```

Reply:
This is mainly my personal preference. I like to always keep my projects one major Release behind because some libraries/frameworks take long to test/cover the latest releases.

Reply:
If you insist on using 3.8 we can do it.

Reply:
I am not sure what my opinion is. I think going for 3.8 is a better choice and shows some freshness, but `django-docker` is not really actively maintained. Not sure if people use it at all?

Reply:
prefers py3.8. But need to change Travis.yml, Dockerfile and Dockerfile.web if opting for 3.8.